packages/expo-router/src/native-tabs/common/elements.tsx

```diff
@@ -90,6 +96,47 @@ export function Icon(props: IconProps) {
   return null;
 }
 
+export interface VectorIconProps<NameT extends string> {
+  /**
+   * The family of the vector icon.
+   *
+   * @example
+   * ```tsx
+   * import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+   * ```
+   */
+  family: {
+    getImageSource: (name: NameT, size: number, color: ColorValue) => Promise<ImageSourcePropType>;
+  };
+  /**
+   * The name of the vector icon.
+   */
+  name: NameT;
+}
+
+/**
+ * Helper component which can be used to load vector icons for `Native Tabs`.
```

Comment:
```suggestion
 * Helper component which can be used to load vector icons for `NativeTabs`.
```

Just a small correction for the component name.